tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1084,6 +1084,27 @@ private void writeSummaryTable(
 
     private Map<String, String> getSortedNames(Organization org, Level desiredLevel) {
         Map<String, String> sortedNames = new TreeMap<>(CLDRConfig.getInstance().getCollator());
+        LocalesWithExplicitLevel cldrLocale =
+                new LocalesWithExplicitLevel(Organization.cldr, desiredLevel);
```

Comment:
This is a **TC user** in the unaffiliated locales, right?

Reply:
It's a user already allowed to access priority items summary.

Reply:
I don't understand this code. The local variable cldrLocale seems to be assigned a value that is never used?

Reply:
@btangmu Thanks! Actually this looks like a copy and paste error from my previous PR, i'll remove it.